etc/dbus-serialbattery/battery.py

```diff
@@ -215,6 +215,10 @@ def manage_charge_voltage_linear(self) -> None:
         voltageSum = 0
         penaltySum = 0
         tDiff = 0
+        
+        PENALTY_BUFFER = 0.010
+        MAX_BATTERY_VOLTAGE = utils.MAX_CELL_VOLTAGE * self.cell_count
+        MIN_BATTERY_VOLTAGE = utils.MIN_CELL_VOLTAGE * self.cell_count
```

Comment:
Maybe it also would make more sense to make the calculations available for all functions in the battery class and round the calculated value like in line 206. What do you think?

Reply:
Thank you for your suggestion. You're absolutely right, it makes more sense to make the calculations available for all functions